packages/utilities/src/diff/WebDiffManager.ts

```diff
@@ -155,9 +155,9 @@ export class WebDiffManager implements IWebDiffManager {
                   fileListToggle: false,
                   fileListStartVisible: false,
                   fileContentToggle: false,
-                  matching: 'lines',
+                  matching: 'none',
                   outputFormat: 'side-by-side',
-                  synchronisedScroll: true,
+                  synchronizedScroll: true,
```

Comment:
I believe this needs to be `synchronisedScroll`.

Reply:
whoops thought i undid this